modules/update-service-overview.adoc

```diff
@@ -39,8 +39,13 @@ Only upgrading to a newer version is supported. Reverting or rolling back your c
 
 During the upgrade process, the Machine Config Operator (MCO) applies the new configuration to your cluster machines. The MCO cordons the number of nodes as specified by the `maxUnavailable` field on the machine configuration pool and marks them as unavailable. By default, this value is set to `1`. The MCO then applies the new configuration and reboots the machine.
 
-If you use {op-system-base-full} machines as workers, the MCO does not update the kubelet because you must update the OpenShift API on the machines first. 
+If you use {op-system-base-full} machines as workers, the MCO does not update the kubelet because you must update the OpenShift API on the machines first.
 
 With the specification for the new version applied to the old kubelet, the {op-system-base} machine cannot return to the `Ready` state. You cannot complete the update until the machines are available. However, the maximum number of unavailable nodes is set to ensure that normal cluster operations can continue with that number of machines out of service.
 
 The OpenShift Update Service is composed of an Operator and one or more application instances.
+
+[NOTE]
+====
+If you have configured the `MachineHealthCheck` resource, ensure that you stop it before updating the cluster. To stop the check, remove the resource.
```

Comment:
Based on the work @slintes has been doing recently, I don't think this is the approach we want to recommend.

We have a pause mechanism that should be used if users want this safety.

That said, I think we are in a better state now and may not have issues with MHCs installed now because of some bug fixes Marc has worked on.

In general, the engineering org seemed to be against pausing MHCs during upgrades as an automatic thing, and in general asked why you would want this. There are cases where MHC can be useful during upgrades (failures can genuinely occur), so perhaps this isn't something we want to be recommending at all?